.vib/fluent-bit/goss/goss.yaml

```diff
@@ -0,0 +1,33 @@
+http://fluent-bit:{{ .Vars.service.ports.http }}/:
+  status: 200
+  body:
+    - /version.*{{ .Env.APP_VERSION }}/
+file:
+  /opt/bitnami/fluent-bit/conf:
+    exists: true
+    filetype: directory
+    mode: "0755"
+  /logs:
+    exists: true
+    filetype: directory
+    mode: "2777"
```

Comment:
Same here. Indeed, you can remove this specific test as you later check for `/logs/cpu.0`, which implicitly guarantees the existence of `/logs`

Reply:
[d884759](https://github.com/bitnami/charts/pull/15453/commits/d88475922fa9ce7829c72589eed76210a1ea9089)